spring-test/src/test/java/org/springframework/test/context/jdbc/DataSourceOnlySqlScriptsTests.java

```diff
@@ -46,7 +46,7 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @ContextConfiguration
-@Sql({ "schema.sql", "data.sql" })
+@Sql({"drop-schema.sql", "schema.sql", "data.sql" })
```

Comment:
We will need to introduce a new, separate test class to test the new feature.

In other words, we need to leave the existing tests 100% unchanged in order to ensure backwards compatibility.